### DISCUSSION TITLE
A documentation typo, probably ?

### DISCUSSION THREAD

**Discussion by m7moudabdel7mid** [livewire/livewire#5973] [4. General]
Hi, i'm not sure if this is the right place for this since v3 docs is not located in livewire/docs.

V3 docs in actions page under "Calling actions from Alpine > Passing parameters" section

```
<div x-data="{ todo: '' }">
    <input type="text" wire:model="todo">
 
    <button x-on:click="$wire.addTodo(todo)">Add Todo</button>
</div>
```

I think this should be like this, right ?

```
    ...
    <input type="text" x-bind:value="todo">
    ...
```

**Comment by PhiloNL** [comment]
Hey @m7moudabdel7mid,
This section of the documentation is specific to:

```blade
<button x-on:click="$wire.addTodo(todo)">Add Todo</button>
```

Where you use Alpine to listen to the click event and call the `addTodo` action, and provide `todo` as a parameter.
So I believe the documentation is correct.

**Reply by m7moudabdel7mid** [reply]
In the example the backend won't get the todo value from the input field 
because the passed value is from alpine data object that will not be updated when the input field updates 
to fix this i think it should be 
`<input type="text" x-model="todo">`
and not 
`<input type="text" wire:model="todo">`

I understand this might not be the point of this section but it will be better if the example actually works, don't you think
